libcst/__init__.py

```diff
@@ -190,8 +190,13 @@
     PartialParserConfig,
 )
 from libcst._removal_sentinel import RemovalSentinel, RemoveFromParent
-from libcst._version import LIBCST_VERSION
 from libcst._visitors import CSTNodeT, CSTTransformer, CSTVisitor, CSTVisitorT
+
+
+try:
+    from libcst._version import version as LIBCST_VERSION
+except ImportError:
+    LIBCST_VERSION = "unknown"
```

Comment:
when can this condition trigger?

Reply:
I think @thatch's comment is meant to be a reply here. 

https://github.com/Instagram/LibCST/pull/471#issuecomment-895529475

> If someone adds it to pythonpath without using setup.py develop or similar.